### PR TITLE
Cw/wrap onno

### PR DESCRIPTION
**Pull request by chriswynn**

### CODE REVIEW

File: src/components/Wrap/index.tsx [hunk 1]
```diff
@@ -1,36 +1,17 @@
...
-        })
-      }
-      {...props}
-    />
-  );
+const wrapDefaultProps: BoxProps = {
+  height: "100%",
+  marginX: "auto",
+  maxWidth: [0, 2, 1],
+  paddingX: 3,
```
[the remark this answers is not in the document]
Comment (chriswynn) [reply src/components/Wrap/index.tsx]: Padding x remains the same no matter what the screen width is

File: src/components/Wrap/index.tsx [hunk 2]
```diff
@@ -1,36 +1,17 @@
...
-          width: "100%",
-        })
-      }
-      {...props}
-    />
-  );
+const wrapDefaultProps: BoxProps = {
+  height: "100%",
+  marginX: "auto",
+  maxWidth: [0, 2, 1],
```
[the remark this answers is not in the document]
Comment (chriswynn) [reply src/components/Wrap/index.tsx]: Null in fact does not work

File: src/lib/theme.ts [hunk 3]
```diff
@@ -32,4 +32,5 @@ export default {
   ],
   lineHeights: [2, 1.5, 1.2],
   maxWidths: [["66.66vw", "55vw"], ["100vw", "90vw"]],
+  breakpoints: [0, 600, 1600],
```
Comment (sethkrasnianski) [inline src/lib/theme.ts]: Where is this used?

Comment (chriswynn) [reply src/lib/theme.ts]: It is automatically used by onno. If you write a style like this `maxWidth: [0, 2, 1]` each value corresponds to each of those breakpoints.

Comment (sethkrasnianski) [reply src/lib/theme.ts]: We need some documentation 😬

Comment (chriswynn) [reply src/lib/theme.ts]: Yeah I definitely need to write some out

**Pull request merged**